spec/api-app-spec.js

```diff
@@ -285,6 +285,7 @@ describe('app module', function () {
     it('should set a badge count', function () {
       app.setBadgeCount(42)
       assert.equal(app.getBadgeCount(), shouldFail ? 0 : 42)
+      app.setBadgeCount(0)
```

Comment:
Maybe this should be done in an `afterEach` to ensure it gets cleared even when the test fails?